pandamium_datapack/data/pandamium/functions/misc/teleport/teleport.mcfunction

```diff
@@ -0,0 +1,15 @@
+data modify storage pandamium:teleport Pos set value [0.,0.,0.]
+
+execute store result storage pandamium:teleport Pos[0] double 1 run scoreboard players get <tp_x> variable
+execute store result storage pandamium:teleport Pos[1] double 1 run scoreboard players get <tp_y> variable
+execute store result storage pandamium:teleport Pos[2] double 1 run scoreboard players get <tp_z> variable
+
+data modify entity @s Pos set from storage pandamium:teleport Pos
+data remove storage pandamium:teleport Pos
+
+#only needs to run if the player stays in the same dimension
+execute as @a[tag=selected_player,x=0] run function pandamium:misc/teleport/anti_tp_exploit
+
+execute positioned as @s run tp @a[tag=selected_player] ~0.5 ~ ~0.5 0 0
```

Comment:
Is there a specific reason why the rotation is set to 0,0?
I think it'd be more natural for it to stay the same

Reply:
Oh wait, this is run as the Minecart, right?
And that's because you can't set Player nbt data, right?

Reply:
Alright, I just did it because the previous teleport function did the same

Reply:
yeah

Reply:
Oh I didn't even realize the previous one did it too xD